cnf-tests/testsuites/e2esuite/gatekeeper/gatekeeper.go

```diff
@@ -695,6 +701,57 @@ var _ = Describe("gatekeeper", func() {
 		})
 	})
 
+	Context("constraints", func() {
+		AfterEach(func() {
+			err := deletePods(testingNamespace, client)
```

Comment:
should this one point to the `constraintValidationNamespace` ?

Reply:
fixed